core/src/main/scala/org/apache/spark/executor/Executor.scala

```diff
@@ -136,6 +136,23 @@ private[spark] class Executor(
   // for fetching remote cached RDD blocks, so need to make sure it uses the right classloader too.
   env.serializerManager.setDefaultClassLoader(replClassLoader)
 
+  private val pluginList = conf.get(EXECUTOR_PLUGINS)
```

Comment:
Just noticed... all this code is in the constructor, so it would be better to not leave unnecessary fields around. Something like:

```
private val plugins: Seq[ExecutorPlugin] = {
  // All this initialization code.
}
```